src/core/PlumedMain.cpp

```diff
@@ -1067,7 +1067,11 @@ void PlumedMain::readInputLine(const std::string & str, const bool& before_init)
     return;
   }
   std::vector<std::string> words=Tools::getWords(str);
-  citations.clear();
```

Comment:
@gtribello what's the reason for this change? Is it related to the PR?

Reply:
I replaced that line with the following:

```c++
if( before_init ) {
    plumed_assert( citations.empty() );
  } else {
    citations.clear();
  }
```

So, the citations.clear is still done in most cases.  I am just not clearing the citations if plumed.init has not been called before the call to readInputLine.  Note readInputLine is used by actions that pass data from the MD code to PLUMED now.  

In other words, I am doing a sanity check here instead to make sure that people are not trying to cite anything.

Reply:
Not trying to cite anything in the interface.  We could change it to call readInputWords instead.  Basically, the problem with shortcuts is that I am calling readInputLine, which clears the citations.  This is the source of the mess.  I added that sanity check in the hope that would prevent folks from making a mess again in the future.